### PR TITLE
Docs: update for OWASP Top Ten 2021

### PR DESCRIPTION
**Pull request by psiinon**
We now have separate pages for 2017 and 2021 with the old URL redirecting to 2021.

I may have forgotten to reference some of the newer add-ons / changes so please double check ;)

Screenshot 2021-12-01 at 15-18-28 ZAPping the OWASP Top 10 (2021)

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by psiinon** [comment]
Like Eval Villain - doh! (meant to add that, will do so now..)

File: site/content/docs/guides/zapping-the-top-10-2021.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+|--|--|--|
+| {{< heading level="5" heading="Common Components" >}}   |  | The 'common components' can be used for pretty much everything, so can be used to help detect all of the Top 10 |
+|   | Manual | [Manipulator-in-the-middle proxy](/docs/desktop/start/features/intercept/)   |
+|   | Manual | [Manual request](/docs/desktop/ui/dialogs/man_req/) / resend   |
+|   | Manual | [Scripts](/docs/desktop/addons/script-console/) |
+|   | Manual | [Search](/docs/desktop/ui/tabs/search/) 
+| [{{< heading level="5" heading="A1 Broken Access Control" >}}](https://owasp.org/Top10/A01_2021-Broken_Access_Control/) | | |
+|   | Automated | Scan Rules tagged with: [OWASP_2021_A01](/alerttags/owasp_2021_a01/)  |
+|   | Automated | [Access Control Testing*](/docs/desktop/addons/access-control-testing/) |
+|   | Manual | [HttpsInfo*](/docs/desktop/addons/https-info/)  |
```
Comment (kingthorin) [inline site/content/docs/guides/zapping-the-top-10-2021.md:35]: Can be dropped

File: site/content/docs/guides/zapping-the-top-10-2021.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+|   | Manual | [Fuzzer](/docs/desktop/addons/fuzzer/), combined with the [FuzzDb*](/docs/desktop/addons/fuzzdb-files/) and [SVN Digger*](/docs/desktop/addons/svn-digger-files/) files |
+| [{{< heading level="5" heading="A4 Insecure Design" >}}](https://owasp.org/Top10/A04_2021-Insecure_Design/) | | |
+|   | Automated | Scan Rules tagged with: [OWASP_2021_A04](/alerttags/owasp_2021_a04/)  |
+| [{{< heading level="5" heading="A5 Security Misconfiguration" >}}](https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2021/Top_10-2021_A5-Security_Misconfiguration)  |  | |
+|   | Automated | Scan Rules tagged with: [OWASP_2021_A05](/alerttags/owasp_2021_a05/)  |
+|   | Manual | [Spider](/docs/desktop/start/features/spider/)  |
+|   | Manual | [Ajax Spider](/docs/desktop/addons/ajax-spider/)   |
+|   | Manual | [Session comparison](/docs/desktop/ui/tlmenu/report/#compare-with-another-session)  |
+|   | Manual | [Access Control Testing*](/docs/desktop/addons/access-control-testing/) |
+|   | Manual | [HttpsInfo*](/docs/desktop/addons/https-info/)  |
```
Comment (kingthorin) [inline site/content/docs/guides/zapping-the-top-10-2021.md:51]: Can be dropped

File: site/content/docs/guides/zapping-the-top-10-2021.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+| [{{< heading level="5" heading="A6 Vulnerable and Outdated_Components" >}}](https://owasp.org/Top10/A06_2021-Vulnerable_and_Outdated_Components/)  |  | |
```
Comment (kingthorin) [inline site/content/docs/guides/zapping-the-top-10-2021.md:52]:
```suggestion
| [{{< heading level="5" heading="A6 Vulnerable and Outdated Components" >}}](https://owasp.org/Top10/A06_2021-Vulnerable_and_Outdated_Components/)  |  | |
```

**Review by kingthorin — APPROVED** [review]

**Comment by psiinon** [comment]
Added Eval Villain, Community Scripts (in the common componenets section) and fixed the first of Rick's issues :)

**Review by kingthorin — APPROVED** [review]
LGTM

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**